kadi/scripts/update_events.py

```diff
@@ -120,30 +160,33 @@ def update(EventModel, date_stop):
     date_stop = DateTime(date_stop)
     cls_name = EventModel.__name__
 
+    # The Update model is used to keep track of last date_stop when the events were
+    # successfully updated.
     try:
+        # Last update for this event model
         update = models.Update.objects.get(name=cls_name)
     except ObjectDoesNotExist:
         logger.info("No previous update for {} found".format(cls_name))
         duration = EventModel.lookback
         update = models.Update(name=cls_name, date=date_stop.date)
         date_start = date_stop - EventModel.lookback
     else:
+        # Duration between last update and current date_stop (usually now)
         duration = date_stop - DateTime(update.date)
         date_start = DateTime(update.date) - EventModel.lookback
         update.date = date_stop.date
 
         # Some events like LoadSegment or DsnComm might change in the database after
         # having been ingested.  Use lookback_delete (less than lookback) to
         # always remove events in that range and re-ingest.
-        if duration >= 0.5 and hasattr(EventModel, "lookback_delete"):
+        if duration > 0 and hasattr(EventModel, "lookback_delete"):
```

Comment:
@taldcroft can you just talk through these changes in the duration for applied lookbacks a bit?

Reply:
This change reflects the hourly processing and the fact that you have new MAUDE data after each comm. So in theory the code can arrive to this point with duration = 2 hrs (2/24), and that means the normal update processing should occur.

What is a little new here is that with the `--maude` flag, there is a pre-condition that there be new MAUDE telemetry for at least one event model in order to get into this function. That's in the `check_maude_server_has_new_telemetry` function called near the top of `main`. 

In normal processing the `duration > 0` is not doing anything, but it might still be necessary for some corner case like giving a stop date that is before the end of available data but there is new MAUDE telemetry.

Reply:
Agreed that the duration stuff is a bit confusing, even to me.